gtk4-macros/tests/template-string.rs

```diff
@@ -35,7 +35,11 @@ mod imp {
         }
     }
 
-    impl ObjectImpl for MyWidget {}
+    impl ObjectImpl for MyWidget {
+        fn dispose(&self, obj: &Self::Type) {
+            obj.first_child().unwrap().unparent();
```

Comment:
dispose could potentially be called multiple times, so I would rather have something like
```rust
if let Some(child) = obj.first_child() {
  child.unparent();
}
```